i18n/pt-BR/docusaurus-plugin-content-docs/current/guides/esp32.md

```diff
@@ -0,0 +1,105 @@
+| PSRAM              | ✔️     | 4MB de SRAM externa                                                        |
+| UART               | ✔️     | Apenas UART0 por enquanto                                                  |
+| I2C                | ✔️     | Somente Master, endereçamento de 10 bits não suportado.                    |
+| I2S                | ❌     |                                                                            |
+| SPI                | ❌     |                                                                            |
+| RMT                | 🟡     | Apenas transmissão, use para controlar NeoPixels                           |
+| PWM                | ❌     |                                                                            |
+| DMA                | ❌     |                                                                            |
+| WIFI               | 🟡     | Escaneamento da rede funciona; [Veja as notas](#simula%C3%A7%C3%A3o-wifi)  |
+| Bluetooth          | ❌     |                                                                            |
+| Timers             | 🟡     |                                                                            |
+| Watchdog           | ❌     |                                                                            |
+| RTC                | 🟡     | Apenas resistores RTC pull-up / pull-down                                  |
+| ADC                | ✔️     | Nota: analogRead() retorna valores até 4095                                |
+| Hall Effect Sensor | ❌     |                                                                            |
+| GDB Debugging      | 🟡     | Apenas atraves [wokwi-gdbserver](https://github.com/wokwi/wokwi-gdbserver) |
+
+Legenda:  
+✔️ - Simulado  
+🟡 - Implementação parcial/trabalho em andamento  
+❌ - Não implementado (mas se você precisar, [abra uma solicitação de recurso](https://github.com/wokwi/wokwi-features/issues/new?labels=enhancement&template=feature_request.md))
+
+## Simulação WiFi
+
+A simulação de WiFi ainda está em andamento. Você pode procurar por redes WiFi, e agora estamos trabalhando na capacidade de nos conectar a redes (associação).
+
+O simulador fornece atualmente dois pontos de acesso WiFi virtuais integrados:
+
+| Nome (SSID) | BSSID             | Descrição                                              |
+| ----------- | ----------------- | ------------------------------------------------------ |
+| Wokwi-GUEST | 42:13:37:55:aa:01 | Abra a rede WiFi (sem necessidade de senha)            |
+| Wokwi-Club  | 42:13:37:55:aa:02 | Rede WiFi somente [assinantes](https://wokwi.com/club) |
+
+A rede **Wokwi-GUEST** pode ser usada por qualquer pessoa e pode acessar um conjunto limitado de serviços da Internet.
+A rede **Wokwi-Club** é limitada para [assinantes](https://wokwi.com/club), e pode acessar todos os servidores da Internet através de um proxy.
```

Comment:
Is assinantes the correct word here? the idea is to refer to club members (who support wokwi)